sdcm/utils/k8s/chaos_mesh_alt.py

```diff
@@ -0,0 +1,59 @@
+# pylint: skip-file
+import logging
+from dataclasses import dataclass
+
+from sdcm.cluster_k8s import KubernetesCluster
+from sdcm.utils.k8s import HelmValues, get_helm_pool_affinity_values
+
+__all__ = ['initialize', 'run_experiment', 'Experiment']
+
+LOGGER = logging.getLogger(__name__)
+NAMESPACE = "chaos-mesh"
+VERSION = "2.5.0"
+HELM_SETTINGS = {
+    'dashboard.create': True,
+    'dnsServer.create': True
+}
+
+
+def initialize(k8s_cluster: KubernetesCluster) -> None:
+    """Installs chaos-mesh on k8s cluster and prepares for future k8s chaos testing."""
+    if k8s_cluster.kubectl(f"get ns {NAMESPACE}", ignore_status=True).ok:
+        LOGGER.info("Chaos Mesh is already installed. Skipping installation.")
+        return
+    LOGGER.info("Installing chaos-mesh...")
+    k8s_cluster.helm("repo add chaos-mesh https://charts.chaos-mesh.org")
+    k8s_cluster.helm('repo update')
+    k8s_cluster.kubectl("create namespace chaos-mesh")
+    k8s_cluster.helm_install(
+        target_chart_name="chaos-mesh",
+        source_chart_name="chaos-mesh/chaos-mesh",
+        version=VERSION,
+        use_devel=False,
+        namespace=NAMESPACE,
+        values=HelmValues(HELM_SETTINGS |
+                          {"chaosDaemon": get_helm_pool_affinity_values(k8s_cluster.POOL_LABEL_NAME,
+                                                                        k8s_cluster.SCYLLA_POOL_NAME)}
+                          ),
+        atomic=True,
+        timeout="30m"
+    )
+    LOGGER.info("chaos-mesh installed successfully.")
+
+
+@dataclass
+class Experiment:
+    """This is where definition of experiments will be crated,
+    different experiments will be inheriting from this class for type checks and common params"""
+
+
+def run_experiment(k8s_cluster: KubernetesCluster, experiment: Experiment) -> None:
```

Comment:
doesn't an experiment have some id or name ?

so we can get it back, get it's status, wait for it to end ?

Reply:
yes, it has name, but I don't know complete answers yet.
We can return some id/name if it will be required and make waiting functions if necessary.